rioxarray/xarray_plugin.py

```diff
@@ -56,8 +54,7 @@ def open_dataset(
         rds = _io.open_rasterio(
             filename_or_obj,
             parse_coordinates=parse_coordinates,
-            chunks=chunks,
-            cache=cache,
```

Comment:
According to the docstring in the first example on the [xarray custom backend documentation](https://docs.xarray.dev/en/stable/internals/how-to-add-new-backend.html), `cache` will not be passed from xarray, and as I understand it xarray will do the caching itself. Hence my decision to not cache here...

Reply:
Thank you for the clarification. I would like to read the document you shared and test out some code before merging so I have a better understanding. Unfortunately, it will likely be a week or so before I get time to do so.

Reply:
No problem, just ping me if you need anything.

Reply:
After reading through the code,  `cache=False` is the correct behavior here. Thank you for catching this.